crates/miden-objects/src/note/mod.rs

```diff
@@ -68,7 +68,7 @@ pub use file::NoteFile;
 /// note's script determines the conditions required for the note consumption, i.e. the target
 /// account of a P2ID or conditions of a SWAP, and the effects of the note. The serial number has
 /// a double duty of preventing double spend, and providing unlikability to the consumer of a note.
-/// The note's inputs allow for customization of its script.
+/// The note's storage values allow for customization of its script.
```

Comment:
In this file, there are ~5 occurences of "inputs" left which still need to be replaced with "note storage" or "storage".

In general, try searching for "inputs" in the directory ./crates/miden-objects/src/note and you'll find more occurences that need to be updated.